v2.1/architecture/replication-layer.md

```diff
@@ -78,6 +78,38 @@ This mechanism lets us avoid tracking leases for every range, which eliminates a
 
 Your table's meta and system ranges (detailed in the distribution layer) are treated as normal key-value data, and therefore have leases, as well. However, instead of using epochs, they have an expiration-based lease. These leases simply expire at a particular timestamp (typically a few seconds)––however, as long as the node continues proposing Raft commands, it continues to extend the expiration of the lease. If it doesn't, the next node containing a replica of the range that tries to read from or write to the range will become the leaseholder.
 
+#### Leaseholder Rebalancing
+
+Because CockroachDB serves reads from a range's leaseholder, it benefits your cluster's performance if the replica closest to the primary geographic source of traffic holds the lease. However, as traffic to your cluster shifts throughout the course of the day, you might want to dynamically shift which nodes hold leases.
+
+To attempt to choose an optimal node as the range's leaseholder, the current leaseholder keeps track of how many requests it receives from each locality as an exponentially weighted moving average, i.e., the more requests that come from a specific locality, the greater the value is, and will transfer the lease if doing so will reduce latency.
+
+Periodically (every 10 minutes by default in large clusters, but more frequently in small clusters), each leaseholder considers whether it should rebalance where the range's replicas are or transfer the lease to another existing replica by considering 3 primary inputs:
+
+	- The number of leases on each node
+	- The number of requests from each locality
+	- The latency between localities
+
+**Intra-locality**
```

Comment:
I'm guessing this is formatted as bold because our h4's show up as small words with thin lines that are hard to see, right?  Might be worth filing a docs site issue with Joe (but totes understood if you just need a bold section now, today).  Ah the joys of the semantic markup lifestyle

Reply:
Yeah, I should just be the change I want to see in the world and take care of this.